daemon/algod/api/client/restClient.go

```diff
@@ -683,6 +683,14 @@ func (client RestClient) SimulateRawTransaction(data []byte) (response model.Sim
 	return
 }
 
+// RawSimulateRawTransaction gets the raw transaction or raw transaction group, and returns relevant simulation results as raw bytes.
+func (client RestClient) RawSimulateRawTransaction(data []byte) (response []byte, err error) {
```

Comment:
nit: do you want to replace the previous `SimulateRawTransaction`?  for I don't think the old implementation get into use any more.  I can modify the log PR on my side tho if you remove the old one.

Reply:
Good point. I was on the fence about this, but I saw that for dryrun we only offer a `RawDryrun` function, so I think it's ok to do the same here. Removed in 4a85c209a